### DISCUSSION TITLE
How to save/export the trained model?

### DISCUSSION THREAD

**Discussion by Schwundfisch** [speechbrain/speechbrain#2553] [Q&A] [answered]
Sorry if this is a stupid question, I might be missing something obvious but I searched for hours and found nothing...
According to this tutorial (https://colab.research.google.com/drive/12bg3aUdr9mTfOGqcB5pSMABoIKPgiwcM), the output folder specified in the hyperparams.yaml and created with sb.create_experiment_directory should include the trained model but it doesn't, even though brain.fit and brain.evaluate terminate with no error. Nor can I find any other method to save the model. Any help please?

**Comment by asumagic** [comment, marked as answer]
The text in the tutorial might be misleading and might need updating, because this test recipe indeed doesn't save anything. Recipes would typically specify a `Checkpointer` to do this, which you can find in pretty much every other recipe.

**Reply by Schwundfisch** [reply]
it worked, thank you so much!
